test/normalize.html

```diff
@@ -0,0 +1,493 @@
+<!DOCTYPE html>
```

Comment:
I thought you meant adding the original test files. I added tests to this pull. Look under `test/cases`/.